### PR TITLE
Search local

### PR DESCRIPTION
**Pull request by newatox**

### CODE REVIEW

File: src/pages/list-contacts/list-contacts.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
       </button>
     </ion-buttons>
   </ion-navbar>
-  <ion-toolbar>
-    <ion-searchbar placeholder={{searchPlaceholder}}></ion-searchbar>
+  <ion-toolbar *ngIf="!(tabRef.getSelected()==tabFrq)">
+    <ion-searchbar [(ngModel)]="searchBarInput" (ionInput)="doSearch($event)" placeholder="Search..."></ion-searchbar>
```
Comment (Cocatrix) [inline src/pages/list-contacts/list-contacts.html]: Please use placeholder defined in code (for i18n)

File: src/pages/list-contacts/list-contacts.ts [hunk 2]
```diff
@@ -93,6 +95,36 @@ export class ListContactsPage {
...
+      this.resetList();
+      return;
+    }
+    console.log(this.searchBarInput);
+    if (currentTab === this.tabAll) this.searchLocalContacts(content, this.contacts);
+    else if (currentTab === this.tabFav) this.searchLocalContacts(content, this.favorites);
+  }
+  searchLocalContacts(content: string, list: Contact[]) {
+    const matchingContacts = [];
+    const regExp = RegExp(content, 'gi');
```
Comment (Cocatrix) [inline src/pages/list-contacts/list-contacts.ts]: Diacritic insensitivity does not exist on class RegExp ?

Comment (newatox) [reply src/pages/list-contacts/list-contacts.ts]: I handled diacritics with a custom RegExp.

File: src/pages/list-contacts/list-contacts.html (hunk 1 above)
Comment (newatox) [reply src/pages/list-contacts/list-contacts.html]: Done.

**Pull request merged**